### DISCUSSION TITLE
Alba has been added to Ruby Toolbox

### DISCUSSION THREAD

**Discussion by trevorturk** [okuramasafumi/alba#239] [General]
Just a note to say Alba has been added now: https://github.com/rubytoolbox/catalog/pull/680

...so hopefully this will help more people find this wonderful gem!

Thank you!

**Comment by okuramasafumi** [comment]
@trevorturk Thank you so much! I've noticed that Alba wasn't there, but I couldn't find a way to add Alba to Ruby Toolbox.
Personally I use Ruby Toolbox often to find new gems, so I'm so glad to see Alba there!

**Comment by trevorturk** [comment]
Another quick note, I asked for a mention in the Ruby Weekly newsletter and they very kindly obliged! https://rubyweekly.com/issues/624

**Reply by okuramasafumi** [reply]
Thank you, Alba got lots of new stargazers recently, probably because of Ruby Weekly!
